src/main/java/com/booking/superride/mapper/RideDetailsMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.booking.superride.mapper;
+
+import com.booking.superride.domain.RideBookingRequest;
+import com.booking.superride.domain.RideDetailsDTO;
+import com.booking.superride.entity.RideDetails;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
+
+@Mapper(componentModel = "spring")
+public interface RideDetailsMapper {
+
+    RideDetailsDTO detailsToDTO(RideDetails rideDetails);
+
+    @Mappings({

```

Comment:
@Mappings is not necessary here, with @Repetable annotation introduced in java 8, @Mapping for all fields are enough